File: Third Year/ISE/Practicas/Práctica 5/writefile.py
#!/usr/bin/env python
# -*- coding:utf-8 -*-

from sys import argv
import subprocess

parametros = argv
filename = str(argv[1])
iterations = str(argv[2])
f = open(filename,'w')
for i in range(0,int(iterations),1):

        f.write(str(i))

File: Third Year/ISE/Practicas/Práctica 4/writefile.pl
#!/usr/bin/perl


use strict;
use warnings;

my $filename = shift @ARGV;

open(my $fh, '>', $filename) or die "Could not open file '$filename' $!";
my $i = 0;
for my $i (0..1000000) {
      print $fh $i;
  }

close $fh;
